middlewares/validators/tasks.js

```diff
@@ -243,6 +245,7 @@ const getUsersValidator = async (req, res, next) => {
           operator: joi.string().valid(Operators.EXCLUDE),
         })
       )
+      .max(20)
```

Comment:
Does 20 have any significance or arbitrary number?

Reply:
Approximate number of public holidays we might observe.